Lib/tomllib/_parser.py

```diff
@@ -290,9 +296,10 @@ def append_nest_to_list(self, key: Key) -> None:
             cont[last_key] = [{}]
 
 
-class Output(NamedTuple):
-    data: NestedDict
-    flags: Flags
+class Output:
+    def __init__(self) -> None:
+        self.data = NestedDict()
+        self.flags = Flags()
```

Comment:
Is anyone likely to rely on the tuple interface here? If so, we could add a `__getitem__()`